packages/mobile/src/walletConnect/walletConnect.ts

```diff
@@ -1,7 +1,39 @@
 import { call } from 'redux-saga/effects'
 import { initialiseWalletConnect } from 'src/walletConnect/saga'
 
+const wcPrefix = 'wc:'
+const deepLinkPrefix = 'celo://wallet/wc?uri='
+const universalLinkPrefix = 'https://valoraapp.com/wc?uri='
```

Comment:
Currently registered universal link on iOS is `vlra.app`
https://github.com/celo-org/wallet/blob/24af3d30f89a6520190a9c78ccc4e62af57b9953/packages/mobile/ios/celo/celo.entitlements#L10
Which is used for firebase dynamics links.

We should add `valoraapp.com` there and also update the associated config on the website: https://valoraapp.co/apple-app-site-association

Reply:
Pinged the web team